packages/container/src/components/FieldPluginContainer.tsx

```diff
@@ -311,7 +313,25 @@ export const FieldPluginContainer: FunctionComponent = () => {
         <AccordionSummary>
           <Typography variant="h3">Content</Typography>
         </AccordionSummary>
-        <AccordionDetails sx={{ position: 'relative' }}>
+        <AccordionDetails
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: '2rem',
+            position: 'relative',
+          }}
+        >
+          <Container
+            sx={{
+              display: 'flex',
+              p: { xs: 0 },
+            }}
+          >
```

Comment:
We could place both the `LanguageView` and the  `ContentView` inside the container. Then we get rid of the `sx` prop on `AccordionDetails`.

We could use the`Stack` component instead of the `Container`, then we only need to set the `gap` property to `6`